pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
> Adding code into the core to support every different approach does not make sense 

This seems to be a good candidate for the core. If we later have an abstraction we can move them behind it, if we really want.

I think that the concerns regarding a technical dept were addressed enough, and the arguments by @davidvossel and by me are explaining pretty well that there is no need to be concerned. So if there are no new arguments, I would appreciate if we could go back to discuss the details of the implementation.